lib/mix/test/mix_test.exs

```diff
@@ -430,6 +430,29 @@ defmodule MixTest do
       System.delete_env("MIX_INSTALL_RESTORE_PROJECT_DIR")
     end
 
+    test "custom compilers", %{tmp_dir: tmp_dir} do
+      Mix.install(
+        [
+          {:install_test, path: Path.join(tmp_dir, "install_test")}
+        ],
+        compilers: [:elixir, :install_test]
+      )
+
+      assert File.dir?(Path.join(tmp_dir, "installs"))
+
+      assert Protocol.consolidated?(InstallTest.Protocol)
+
+      assert_received {:mix_shell, :info, ["==> install_test"]}
+      assert_received {:mix_shell, :info, ["Compiling 3 files (.ex)"]}
+      assert_received {:mix_shell, :info, ["Generated install_test app"]}
+      assert_received {:mix_shell, :info, ["==> mix_install"]}
```

Comment:
We always print the app name before we print anything within the app. This is expected.

Reply:
Alright, I just wondered because it doesn't print the mix_install app in the other tests